### DISCUSSION TITLE
Appcenter elementary does not work

### DISCUSSION THREAD

**Discussion by vento124** [elementary/appcenter#1930] [Q&A]
Yesterday I downloaded Elementary  6.1 the App center before doing the first update was working fine after the update and reboot since then when I click on it does not open a round black circle keeps going round and round  after that it stops and nothing happens  I must so many times it just does not open. Any help please

**Comment by vento124** [comment]
Can anybody help me thanks on this issue

**Reply by mayankjr98** [reply]
I'm also facing same problem.**I think this will fix with new update.**

**Comment by lenemter** [comment]
Can you launch AppCenter from Terminal and send terminal output here if there is any, please
```
killall io.elementary.appcenter
io.elementary.appcenter
```

**Reply by yvpic** [reply]
I got 
> (io.elementary.appcenter:3860): GLib-CRITICAL **: 21:52:21.132: g_strsplit: assertion 'string != NULL' failed
Erreur de segmentation

**Reply by danirabbit** [reply]
@yvpic can you follow the directions here and post the output of the stack trace? That would help to figure out where the crash is happening so we can deploy a fix: https://docs.elementary.io/contributor-guide/feedback/reporting-issues/inspecting-crashes

**Reply by yvpic** [reply]
Hello, it's an infinite list of (io.elementary.appcenter:11913): Handy-CRITICAL **: 18:16:54.085: hdy_carousel_box_get_nth_child: assertion 'n < hdy_carousel_box_get_n_pages (self)' failed

**Reply by yvpic** [reply]
and before I got, (io.elementary.appcenter:3368): Handy-CRITICAL **: 18:26:17.136: hdy_carousel_box_get_nth_child: assertion 'n < hdy_carousel_box_get_n_pages (self)' failed

(io.elementary.appcenter:3368): Handy-CRITICAL **: 18:26:17.136: hdy_carousel_box_scroll_to: assertion 'GTK_IS_WIDGET (widget)' failed
 and

* (io.elementary.appcenter:3368): WARNING **: 18:27:09.746: ScreenshotCache.vala:192: Error getting modification time of remote screenshot file: Opération non prise en charge

** (io.elementary.appcenter:3368): WARNING **: 18:27:09.746: ScreenshotCache.vala:192: Error getting modification time of remote screenshot file: Opération non prise en charge

** (io.elementary.appcenter:3368): WARNING **: 18:27:09.746: ScreenshotCache.vala:192: Error getting modification time of remote screenshot file: Opération non prise en charge

** (io.elementary.appcenter:3368): WARNING **: 18:27:09.746: ScreenshotCache.vala:231: Unable to download screenshot from io/elementary/appcenter/df3ac5e774bb38595c2340e79d8f9d6b/screenshots/image-1_1248x777.png: Opération non prise en charge

** (io.elementary.appcenter:3368): WARNING **: 18:27:09.746: ScreenshotCache.vala:231: Unable to download screenshot from io/elementary/appcenter/df3ac5e774bb38595c2340e79d8f9d6b/screenshots/image-2_1248x777.png: Opération non prise en charge

** (io.elementary.appcenter:3368): WARNING **: 18:27:09.746: ScreenshotCache.vala:231: Unable to download screenshot from io/elementary/appcenter/df3ac5e774bb38595c2340e79d8f9d6b/screenshots/image-3_orig.png: Opération non prise en charge

No stack with backtrack.

The problem ramins : AppCenter doesn't launch.

Thanx!

**Comment by GeneralAccess** [comment]
All I get is the rotating circle then nothing, so I tried the 2 commands suggested above, with the following result, appcenter starts and is usable, but the terminal shows these errors.:
```
chris@desktop:~$ killall io.elementary.appcenter
chris@desktop:~$ io.elementary.appcenter

(io.elementary.appcenter:3555): PackageKit-WARNING **: 22:18:51.917: failed to set proxy: GDBus.Error:org.gtk.GDBus.UnmappedGError.Quark._pk_2dengine_2derror_2dquark.Code3: setting the proxy failed: failed to get the uid

** (io.elementary.appcenter:3555): WARNING **: 22:18:51.917: PackageKitBackend.vala:228: Unable to set proxy for PackageKit to use: GDBus.Error:org.gtk.GDBus.UnmappedGError.Quark._pk_2dengine_2derror_2dquark.Code3: setting the proxy failed: failed to get the uid

(io.elementary.appcenter:3555): Handy-CRITICAL **: 22:19:08.677: hdy_carousel_box_get_nth_child: assertion 'n < hdy_carousel_box_get_n_pages (self)' failed

(io.elementary.appcenter:3555): Handy-CRITICAL **: 22:19:08.677: hdy_carousel_box_scroll_to: assertion 'GTK_IS_WIDGET (widget)' failed

(io.elementary.appcenter:3555): Handy-CRITICAL **: 22:19:13.677: hdy_carousel_box_get_nth_child: assertion 'n < hdy_carousel_box_get_n_pages (self)' failed

(io.elementary.appcenter:3555): Handy-CRITICAL **: 22:19:13.677: hdy_carousel_box_scroll_to: assertion 'GTK_IS_WIDGET (widget)' failed

(io.elementary.appcenter:3555): Handy-CRITICAL **: 22:19:18.681: hdy_carousel_box_get_nth_child: assertion 'n < hdy_carousel_box_get_n_pages (self)' failed

(io.elementary.appcenter:3555): Handy-CRITICAL **: 22:19:18.681: hdy_carousel_box_scroll_to: assertion 'GTK_IS_WIDGET (widget)' failed

(io.elementary.appcenter:3555): Handy-CRITICAL **: 22:19:23.684: hdy_carousel_box_get_nth_child: assertion 'n < hdy_carousel_box_get_n_pages (self)' failed

(io.elementary.appcenter:3555): Handy-CRITICAL **: 22:19:23.684: hdy_carousel_box_scroll_to: assertion 'GTK_IS_WIDGET (widget)' failed

(io.elementary.appcenter:3555): Handy-CRITICAL **: 22:19:28.685: hdy_carousel_box_get_nth_child: assertion 'n < hdy_carousel_box_get_n_pages (self)' failed

(io.elementary.appcenter:3555): Handy-CRITICAL **: 22:19:28.685: hdy_carousel_box_scroll_to: assertion 'GTK_IS_WIDGET (widget)' failed


```
It now works from the icon on the Plank. However trying to do an update results in:
```
Failed to update operating system updates
This may have been caused by sideloaded or manually compiled software, a third-party software source, or a package manager error. Manually refreshing updates may resolve the issue.

E: https://ppa.launchpadcontent.net/elementary-os/stable/ubuntu jammy/main amd64 appstream-data-pantheon all 5.0+r300~ubuntu7.1.1 is not (yet) available (404  Not Found [IP: 2620:2d:4000:1::3e 443])
E: https://ppa.launchpadcontent.net/elementary-os/stable/ubuntu jammy/main amd64 appstream-data-pantheon-icons all 5.0+r300~ubuntu7.1.1 is not (yet) available (404  Not Found [IP: 2620:2d:4000:1::3e 443])
E: https://ppa.launchpadcontent.net/elementary-os/stable/ubuntu jammy/main amd64 appstream-data-pantheon-icons-hidpi all 5.0+r300~ubuntu7.1.1 is not (yet) available (404  Not Found [IP: 2620:2d:4000:1::3e 443])
E: http://archive.ubuntu.com/ubuntu jammy-security/main amd64 linux-headers-generic-hwe-22.04 amd64 6.2.0.35.35~22.04.13 is not (yet) available (404  Not Found [IP: 2620:2d:4002:1::102 80])
E: http://archive.ubuntu.com/ubuntu jammy-security/main amd64 linux-image-generic-hwe-22.04 amd64 6.2.0.35.35~22.04.13 is not (yet) available (404  Not Found [IP: 2620:2d:4002:1::102 80])
```
This also seems to work the second time I pressed Update.
